.changeset/selfish-books-fail.md

```diff
@@ -0,0 +1,6 @@
+---
+"@comet/admin-date-time": major
+"@comet/admin": major
```

Comment:
This is not specific to the `admin-date-time` package. 

```suggestion
"@comet/admin": major
```

Reply:
Also, I'd say this is a minor change.